### ISSUE TITLE
Consider always megazording

### ISSUE THREAD

**Issue by thomcc** [mozilla/application-services#558]
In https://github.com/mozilla/application-services/pull/557, I linked to a test I ran on the sizes of various builds (reproduced here with irrelavant parts cut for convenience):

<img width="563" alt="screen shot 2019-01-17 at 11 22 52 am" src="https://user-images.githubusercontent.com/860665/51344382-602ee200-1a4d-11e9-91f2-e3cf98c98425.png">

It's worth noting that for Os and Oz, `libreference_browser.so` (which includes everything) is smaller than `libplaces_ffi.so` or `liblogins_ffi.so`. it's also only 1MB larger than `libfxaclient_ffi.so`, and 400k larger than `liblockbox.so`.

Basically, I'm not sure these options and build complexity are worth the trouble. This might become less true over time (especially if whatever rustc bug that's causing this is fixed), but... yeah, at the moment it's certainly not buying us much.

@ncalexan, thoughts?

**Comment by rfk** [comment]
Closing as dupe of https://github.com/mozilla/application-services/issues/876
